modules/@apostrophecms/db/index.js

```diff
@@ -124,9 +135,23 @@ module.exports = {
         };
         self.apos.dbClient = await mongo.MongoClient.connect(uri, connectOptions);
         const parsed = new URL(uri);
+        self.uri = uri;
         self.apos.db = self.apos.dbClient.db(parsed.pathname.substr(1));
       },
+      async versionCheck() {
+        if (!self.options.versionCheck) {
+          return;
+        }
+        const oldGlobal = await self.apos.db.collection('aposDocs').findOne({
+          type: 'apostrophe-global'
+        });
+        if (oldGlobal) {
+          throw new Error(`There is a problem with the database: ${self.uri ? (self.uri + ' ') : ''}
 
+This database already contains an Apostrophe 2.x website. Exiting to avoid
+causing content loss. Content migration tools from 2.x to 3.x are planned.`);
```

Comment:
Let's not include this last sentence. It dates the error message and code like this isn't where we should be discussing plans.

Reply:
@localghost443 wanted to sign off on the language too so I'm requesting review from him so we can change this once rather than twice.

Reply:
"There is a problem with the database: mongodb://localhost:27017/whatever

This database contains an Apostrophe 2.x website. Exiting to avoid content loss."

Removed the last sentence and tweaked slightly for brevity.